app/controllers/alchemy/admin/sites_controller.rb

```diff
@@ -1,6 +1,17 @@
 module Alchemy
   module Admin
     class SitesController < ResourcesController
+      def select
+        authorize! :select, :alchemy_admin_site
+
+        session[:site_id] = requested_site_id
+        do_redirect_to params[:redirect_to]
+      end
+
+      private
```

Comment:
Style/EmptyLinesAroundAccessModifier: Keep a blank line before and after `private`.